en/config_fw/weight_and_altitude_tuning.md

```diff
@@ -1,52 +1,63 @@
 
-[WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_BASE) specifies the actual weight of the vehicle at any given time.
-This parameter can be set to reflect, for example, a payload attached to the vehicle that was not present during the tuning phase.
+You can determine the values by measuring the weight of the vehicle using a scale in the tuning configuration and when flying with a payload.
 
-When both [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE) and [WEIGHT_GROSS](../advanced_config/parameter_reference.md#WEIGHT_GROSS) are set to a number larger than `0`, PX4 will use this information to scale the maximum climb rate, minimum sink rate, and adjust airspeed limits according to the weight ratio.
-The interested reader is referred to [the note section](../config_fw/weight_and_altitude_tuning.md#notes-to-the-derivation-of-weight-and-density-compensation) below for further details.
+Scaling is performed when _both_ `WEIGHT_BASE` and `WEIGHT_GROSS` are greater than `0`, and will have no effect if the values are the same.
+See the [algorithms](#weight-and-density-compensation-algorithms) section below for more information.
 
-## Compensation for Air Density
+## Air Density Compensation
 
 ### Specify a Service Ceiling
+
 In PX4 the service ceiling [FW_SERVICE_CEIL](../advanced_config/parameter_reference.md#FW_SERVICE_CEIL) specifies the altitude in standard atmospheric conditions at which the vehicle is still able to achieve a maximum climb rate of 0.5 m/s at maximum throttle and weight equal to [WEIGHT_BASE](../advanced_config/parameter_reference.md#WEIGHT_BASE).
 By default this parameter is disabled and no compensation will take place.
-This parameter needs to be estimated experimentally. It is always better to set a conservative value (lower value) than an optimistic value.
+
+This parameter needs to be determined experimentally.
+It is always better to set a conservative value (lower value) than an optimistic value.
 
 ### Apply Density Correction to Minimum Sink Rate
-During the initial tuning the minimum sink rate is set using [FW_T_SINK_MIN](../advanced_config/parameter_reference.md#FW_T_SINK_MIN).
 
-If the initial tuning is not done in standard sea level conditions then [FW_T_SINK_MIN](../advanced_config/parameter_reference.md#FW_T_SINK_MIN) needs to be multiplied with correction factor P:
+The minimum sink rate is set in [FW_T_SINK_MIN](../advanced_config/parameter_reference.md#FW_T_SINK_MIN).
+
+If the [Basic TECS tuning](../config_fw/position_tuning_guide_fixedwing.md#tecs-tuning-altitude-and-airspeed) was not done in standard sea level conditions then the [FW_T_SINK_MIN](../advanced_config/parameter_reference.md#FW_T_SINK_MIN) parameter must be modified by multiplying with correction factor $P$ (where $\rho$ is the air density during tuning):
 
 $$P = \sqrt{\rho\over{\rho_{sealevel}}}$$
```

Comment:
So the density has to be known at tuning time right? Doesn't that mean you'll have to re-tune to set this because you probably didn't measure it?

Should we suggest us 1.225 kg/m^3 for the sea level value. How do you go about measuring the density when you're tuning? Can we get this from any of our on-board sensors? 

How sensitive is vehicle behaviour to getting this wrong?

Reply:
@RomanBapst I'm going to stop chasing you on these.